src/libs/Firebase/types.ts

```diff
@@ -21,4 +21,7 @@ type FirebaseAttributes = {
     policyRole: string;
 };
 
-export type {StartTrace, StopTrace, TraceMap, Log, FirebaseAttributes};
+// TODO confirm which attributes are required for Firebase
+type FirebaseAttributes = Pick<PerfAttributes, 'accountId' | 'personalDetailsLength' | 'reportActionsLength' | 'reportsLength' | 'policiesLength'>;
```

Comment:
NAG not sure how easy it is to clean up but would be nice to have `accountID`
```suggestion
type FirebaseAttributes = Pick<PerfAttributes, 'accountID' | 'personalDetailsLength' | 'reportActionsLength' | 'reportsLength' | 'policiesLength'>;
```

Reply:
It will be easy to make this change; however, the current dashboard already uses "accountId" as a field. Introducing "accountID" as an additional field could complicate data analysis. For example, when searching for a specific ID, we would need to check both "accountId" and "accountID," which may lead to inconsistencies and make querying a bit harder.